.github/ISSUE_TEMPLATE.md

```diff
@@ -1,41 +1,19 @@
-Please read the following information before posting the issue.
-
-## Before posting the issue...
-
-* If you're having trouble with the NuGet.org Website, please post in [NuGetGallery issues](http://github.com/nuget/nugetgallery/issues)
-* If you're having trouble with the NuGet client tools (the Visual Studio extension, NuGet.exe command line tool, etc.), you are in the right place.
-
-Remove the content above here and fill out details below.
+<!-- Please read the following information before posting the issue. -->
+<!-- If you're having trouble with the NuGet client tools (the Visual Studio extension, NuGet.exe command line tool, etc.), you are in the right place. -->
+<!-- If you're having trouble with the NuGet.org Website, please post in [NuGetGallery issues](http://github.com/nuget/nugetgallery/issues) -->
 
 ## Details about Problem
 
-NuGet product used (NuGet.exe | VS UI | Package Manager Console | dotnet.exe):
-
-NuGet version (x.x.x.xxx):
-
-dotnet.exe --version (if appropriate):
-
-VS version (if appropriate):
+NuGet product used (NuGet.exe | VS | MSBuild | dotnet.exe):
```

Comment:
Maybe sort these from most likely to least likely? 

`NuGet.exe | dotnet.exe | VS | MSBuild`

Reply:
I don't have a preference here.

I'm not sure I agree that order is the most likely on, I think there's more incoming VS issues than dotnet ones right now.  

I was strongly debating removing that, but I think it provides value of giving you a sense that you are in the right repo.